tests/system/test_system.py

```diff
@@ -1814,12 +1846,116 @@ def test_count_query_in_transaction(client, cleanup):
 
     with pytest.raises(ValueError) as exc:
         create_in_transaction_helper(transaction, client, collection_id, cleanup)
-        assert exc.exc_info == "Collection can't have more than 2 documents"
+    assert str(exc.value) == "Collection can't have more than 2 docs"
 
     collection = client.collection(collection_id)
 
-    query = collection.where("a", "==", 1)
+    query = collection.where(filter=FieldFilter("a", "==", 1))
     count_query = query.count()
     result = count_query.get()
     for r in result[0]:
         assert r.value == 2  # there are still only 2 docs
+
+
+def test_query_with_and_composite_filter(query_docs):
```

Comment:
The test cases below shows how to build composite filters using And and Or and how to pass them to the `filter` argument in `where`.